mycli/sqlcompleter.py

```diff
@@ -1010,6 +1010,9 @@ def extend_columns(self, column_data, kind):
 
         metadata = self.dbmetadata[kind]
         for relname, column in column_data:
+            if relname not in metadata[self.dbname]:
+                _logger.error("relname '%s' was not found in db '%s'", relname, self.dbname)
+                continue
```

Comment:
Can you tell me why this check is needed here? Perhaps you can add a comment with the reason so we don't remove it in the future.

Reply:
this was the exact spot in the code where exception was bring thrown:
```
KeyError: '_vt_hold_cb11180fb29d11efbe790affca8db0b9_20241206001421'
```
if we do the fix in the other file, then this isn't necessary, although i feel a more informative error with the stacktrace is much better. this whole thing was harder to debug lacking this, so i added it first to figure out the rest.

Reply:
added a comment